src/modules/adaptors/Morpho/MorphoBlue/MorphoBlueCollateralAdaptor.sol

```diff
@@ -0,0 +1,230 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.21;
+
+import { BaseAdaptor, ERC20, SafeTransferLib, Cellar, PriceRouter, Math } from "src/modules/adaptors/BaseAdaptor.sol";
+import { MorphoBlueHealthFactorLogic } from "src/modules/adaptors/Morpho/MorphoBlue/MorphoBlueHealthFactorLogic.sol";
+import { IMorpho, MarketParams, Id } from "src/interfaces/external/Morpho/MorphoBlue/interfaces/IMorpho.sol";
+
+/**
+ * @title Morpho Blue Collateral Adaptor
+ * @notice Allows addition and removal of collateralAssets to Morpho Blue pairs for a Cellar.
+ * @dev This adaptor is specifically for Morpho Blue Primitive contracts.
+ *      To interact with a different version or custom market, a new
+ *      adaptor will inherit from this adaptor
+ *      and override the interface helper functions. MB refers to Morpho
+ *      Blue
+ * @author crispymangoes, 0xEinCodes
```

Comment:
If you want you can flip the names, or even remove my name entirely, since this is all your work, and I am just reviewing it :)

Reply:
Haha it doesn't really matter to me, but I'll flip them in case people have questions and can direct them to me first.